content/backend/graphql-js/0-introduction.md

```diff
@@ -16,20 +16,20 @@ GraphQL is the rising star of backend technologies. It replaces REST as an API d
-  * ... provides an editor where you can write queries, mutations & subscriptions, with auto-completion(!) and syntax highlighting.
-  * ... lets you easily share your API operations.
+* [`graphql-yoga`](https://github.com/prisma/graphql-yoga): A fully-featured GraphQL server with a focus on easy setup, performance & great developer experience. It is built on top of [Express](https://expressjs.com/), [`apollo-server`](https://github.com/apollographql/apollo-server), [`graphql-js`](https://github.com/graphql/graphql-js) and more.
+* [Prisma](https://www.prisma.io/): Replaces traditional ORMs. Use the Prisma client to implement your GraphQL resolvers and simplify database access.
+* [GraphQL Playground](https://github.com/prisma/graphql-playground): A "GraphQL IDE" that allows you to interactively explore the functionality of a GraphQL API by sending queries and mutations to it. It's somewhat similar to [Postman](https://www.getpostman.com/) which offers comparable functionality for REST APIs. Among other things, a GraphQL Playground:
+  * Auto-generates comprehensive documentation for all available API operations.
+  * Provides an editor where you can write queries, mutations & subscriptions, with auto-completion(!) and syntax highlighting.
+  * Lets you easily share your API operations.
 
 ### What to expect
 
 The goal of this tutorial is to build an API for a [Hacker News](https://news.ycombinator.com/) clone. Here is a quick rundown of what to expect in this tutorial.
 
-You'll start by learning the basics of how a GraphQL server works, simply by defining a [_GraphQL schema_](https://www.prisma.io/blog/graphql-server-basics-the-schema-ac5e2950214e) for the server and writing corresponding _resolver functions_. In the beginning, these resolvers will only work with data that's stored in-memory - so nothing will be persisted beyond the runtime of the server.
+You'll start by learning the basics of how a GraphQL server works, simply by defining a [_GraphQL schema_](https://www.prisma.io/blog/graphql-server-basics-the-schema-ac5e2950214e) for the server and writing corresponding _resolver functions_. In the beginning, these resolvers will only work with data that's stored in-memory - so nothing will persist beyond the runtime of the server.
 
-Because nobody wants a server that's not able to store and persist data, you're going to add a database layer to it. The database layer is powered by [Prisma](https://www.prisma.io/) and will be connected to your GraphQL server via the [Prisma client](https://www.prisma.io/docs/prisma-client). 
+Because nobody wants a server that's not able to store and persist data, you're going to add a database layer to it. The database layer is powered by [Prisma](https://www.prisma.io/) and will be connected to your GraphQL server via the [Prisma client](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client/api#overview). 
```

Comment:
can you adjust "the Prisma client" to just "Prisma Client", incl removing the article? (here and wherever else you come across it 🙂 )

Reply:
Done! ✅🙂